src/index.html

```diff
@@ -2,12 +2,25 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta
+    name="viewport"
+    content="
+      width=device-width,
+      initial-scale=1.0
+    "
+  >
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Antisnake</title>
   <link rel="stylesheet" href="./styles/main.css">
 </head>
 <body>
-  <h1>Antisnake</h1>
+  <div class="container">
+    <div class="container__item item--1">1</div>
```

Comment:
I guess, we should use modifiers for our elements.
```suggestion
    <div class="container__item containter__item--1">1</div>
```
or 
```suggestion
    <div class="container__item container__item-1">1</div>
```

Reply:
Так, я згоден. Не виспався учора, от і, невірно записав.
Визнаю.

Reply:
✓